source/account_structure/index.html.md.erb

```diff
@@ -1,13 +1,13 @@
 ---
-title: Account structure
+title: How accounts work
 last_reviewed_on: 2020-06-29
 review_in: 6 months
-weight: 95
+weight: 40
 ---
 
-# Account structure
+# How accounts work
 
-You can read more about [reporting](/reporting/#reporting).
+You can read more about [reporting](/reporting/#report-on-a-payment).
```

Comment:
this feels a bit odd here - maybe it's a hangover from a previous structure?

Reply:
Also the subheadings here 'editing Pay services' and 'editing PSP serivces' are a bt confusing. Maybe it's something about managing settings in Pay or managing settings on your PSP.

Reply:
agree, will change